source/overloading.tex

```diff
@@ -1697,6 +1760,18 @@
 \end{example}
 or, if not that,
 
+\item
+\tcode{F1} and \tcode{F2} are generated
+from class template argument deduction\iref{over.match.class.deduct}
+for a class \tcode{D}, and
+\tcode{F2} is generated
+from inheriting constructors from a base class of \tcode{D}
+while \tcode{F1} is not, and
+for each explicit function argument,
+the corresponding parameters of \tcode{F1} and \tcode{F2}
+are either both ellipses or have the same type,
+or, if not that,
+
```

Comment:
This seems to be inserted between the wrong items.

Reply:
fixed, thanks!